gui/src/components/HeaderNavBar.tsx

```diff
@@ -8,13 +8,13 @@ export const HeaderNavBar = () => {
       <header role="banner" className="boss-header usa-header usa-header--basic">
         <div className="usa-banner">
           <img
-            src="doj-logo.svg"
+            src="doj-logo.png"
             alt="U.S. Trustee Program banner"
             className="doj-logo usa-banner__header"
           ></img>
           <div className="site-title">
             U.S. Trustee Program
-            <span className="sub-title">Bankruptcy Oversight Support System</span>
+            <span className="sub-title">Case Management System (CAMS)</span>
```

Comment:
@jayaneti1, Stan had stated the title is `CAse Management System`, complete with the capital A to build out the acronym. Should that be reflected here?

Reply:
Yes Please

Reply:
```suggestion
            <span className="sub-title">CAse Management System (CAMS)</span>
```
To keep the title casing correct, even though it's not _title cased_.